compiler/include/symbol.h

```diff
@@ -462,6 +462,8 @@ class ModuleSymbol : public Symbol {
   void                 moduleUseAdd(ModuleSymbol* module);
   void                 moduleUseRemove(ModuleSymbol* module);
 
+  bool                 isVisible(BaseAST* scope);
```

Comment:
I wonder if isVisible() could be declared const i.e. bool isVisible(BaseAST* scope) const?  I scanned
isVisible() and sense it should be possible but am not sure whether doing so would require a ripple of
const.